src/mscorlib/src/System/PinnableBufferCacheEventSource.cs

```diff
@@ -0,0 +1,74 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics.Tracing;
+
+namespace System
+{
+    [EventSource(Guid = "38ed3633-5e3f-5989-bf25-f0b1b3318c9b", Name = "Microsoft -DotNETRuntime-PinnableBufferCache-System")]
+    internal sealed class PinnableBufferCacheEventSource : EventSource
+    {
+        public static readonly PinnableBufferCacheEventSource Log = new PinnableBufferCacheEventSource();
+
```

Comment:
Also need to add a private .ctor that takes the Guid directly or it Jits up a mass of reflection on startup (which takes time):
```csharp
private PinnableBufferCacheEventSource() : base(new Guid(0x38ed3633, 0x5e3f, 0x5989, 0xbf, 0x25, 0xf0, 0xb1, 0xb3, 0x31, 0x8c, 0x9b)
```

Reply:
Done